projects/wave-core-new/src/lib/users/user.service.ts

```diff
@@ -207,6 +207,26 @@ export class UserService {
 
         return of(session);
     }
+
+    oidcInit() : Observable<AuthCodeRequestURL> {
+        return this.backend.oidcInit();
+    }
```

Comment:
This call is what you want to avoid, right? You had some TODO in the backend PR.

Reply:
I don't think this call can be avoided. It would be nice to get rid of the tryLogin call in the AppComponent by sending an answer directly to the backend, but I don't know how the backend would get the frontend back.